auction-app-frontend/src/components/CheckoutPaymentForm/CheckoutPaymentForm.jsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from "react";
+import { useStripe, useElements, CardNumberElement, CardExpiryElement, CardCvcElement } from '@stripe/react-stripe-js';
+
+import { LoadingComponent, Button, ErrorComponent } from "src/components";
+import { BUTTON_VARIANTS, BUTTON_LABELS } from "src/constants";
+
+import "./style.scss";
+
+const CheckoutPaymentForm = ({ clientSecret, onPaymentSuccess }) =>
+{
+    const [errorMessage, setErrorMessage] = useState("");
```

Comment:
Is there any other way to handle those errors
Can we have an object with errors, or something like that, to avoid having 5 error states